ociinstaller/versionfile/versionfile.go

```diff
@@ -68,16 +69,29 @@ func read(path string) (*VersionFile, error) {
 	var data VersionFile
 
 	if err := json.Unmarshal([]byte(file), &data); err != nil {
-		fmt.Println("---- error: ", err)
+		log.Println("Error while reading version file", err)
```

Comment:
`log` looked like a better idea than printing out onto the console. Albeit - forgot to put in the log level

Reply:
we return an error - this will get printed on the screen anyway right?

I think we need to look at the whole issue of logging as one, rather than just adding as few random lines here and there